app/routes/auth.py

```diff
@@ -29,8 +32,12 @@ def login_for_access_token(
     access_token = create_access_token(
         data={"sub": username}, expires_delta=access_token_expires
     )
-    return {"access_token": access_token, "token_type": "bearer"}
-
+    return {"access_token": access_token, 
+            "token_type": "bearer",
+            "is_admin": is_admin,
+            "is_superadmin": is_superadmin,
+            }
```

Comment:
No devolver el is admin y el is superadmin eso ya se puede deducir del sub del tokeb